src/App.vue

```diff
@@ -230,7 +230,9 @@ export default class App extends Mixins(
 
   // configuration store references
   @Getter(useConfigurationStore) getAuthApiUrl!: string
+  // @Getter(useConfigurationStore) getBusinessDashUrl!: string
```

Comment:
This getter is used in this module but if I uncomment this then I get a warning that it's already imported in a mixin. Still, it's nice to see, here, where this is imported from.